services/docs/lib/services/document.js

```
@@ -7,19 +7,72 @@ const permissions = require('../modules/permissions');
 const { ObjectId } = require('mongodb');
 
 const createError = require('http-errors');
+const { getDocumentLockServiceOptions } = require('../modules/serviceOptions');
 
 // Helper functions
 const getDocUsersList = doc => Object.keys(doc ? doc.users : []).map(k => doc.users[k]);
 const getRequestedStatesFromQuery = (resource, query) => {
   return query.states ? query.states.split(',') : Object.keys(resource.states);
 };
 
+module.exports.deleteLock = async function deleteLocks(id, user, editLock, db) {
+  if (!id) {
+    return undefined;
+  }
+
+  const match = { _id: ObjectId(id) };
```

I would also check the id validity with `createObjectId` and throw an error if invalid